src/main/scala/Api.scala

```diff
@@ -82,14 +85,34 @@ class Api(kafkaClientActorRef: ActorRef)
 
   def healthCheck: StandardRoute = {
     getHealth match {
-      case Health(true, message, None) => complete(Json.toJson(message).toString())
+      case Health(true, message, None) => complete(message)
       case Health(false, _, Some(e)) => failWith(e)
     }
   }
 
   def getHealth: Health = {
     val health = healthCheck("Health") {
-      Result.healthy("OK")
+
+      val checkDuration = timeoutDuration.duration
+      val healthFuture = askFor[DescribeClusterResult](DescribeKafkaCluster).map { clusterDesc =>
+          val clusterNodes = clusterDesc.nodes.get(checkDuration.length, checkDuration.unit)
+          val clusterId = clusterDesc.clusterId.get(checkDuration.length, checkDuration.unit)
+          logger.debug(s"clusterId: $clusterId; nodes: $clusterNodes")
+
+          val clusterIdAvailable: Boolean = Option(clusterId).isDefined
+          if (clusterIdAvailable && !clusterNodes.isEmpty) {
+            Result.healthy("OK")
```

Comment:
was thinking of returning cluster info instead of just OK

Reply:
Just implemented it, however is there a way to pass an **unescaped** JSON string to the Result? 

Right now, I get 

```http
HTTP/1.1 200 OK
Content-Length: 139
Content-Type: application/json
Date: Sat, 05 Jan 2019 08:45:29 GMT
Server: akka-http/10.0.6

"{\"cluster_id\":\"euj63DLESi2myvKrrc_i5A\",\"nodes\":[{\"id\":0,\"id_string\":\"0\",\"host\":\"10.0.0.40\",\"port\":9092,\"rack\":null}]}"


```

Reply:
If you look at JsonOps.scala here is how json is marshalled if you had a case class. If you look at Api.scala you will see how this is used, using the `.map(Json.toJson(_))`. I think you removed this part

Reply:
I removed it because `OK` was becoming `"OK"`. The response shouldn't have had extra quotes to begin with. 

Anyways, same result with 

```scala
    getHealth match {
      case Health(true, message, None) => complete(Json.toJson(message))
      case Health(false, _, Some(e)) => failWith(e)
    }
```

I wish these docs had some example 😞 https://metrics.dropwizard.io/3.1.0/manual/json/

Reply:
Create the marshaller in JsonOps.scala and an Object for these values to fit into.